config.json

```diff
@@ -779,7 +779,7 @@
         "name": "Acronym",
         "uuid": "440d78d1-9dea-466f-9bd4-935eed067409",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["strings", "arrays"],
```

Comment:
Not sure what kind of solutions you have in mind here. Most solutions I have seen use strings, regular expressions and array transformations (`map`).

Reply:
Ya, that sounds right to me now that I see it, will make the change.